www/js/util.js

```diff
@@ -1649,6 +1672,10 @@ function addChatMessage(data) {
 
     pingMessage(isHighlight);
 
+    if (USEROPTS.notifications && document.hidden)
+    {
+        new Notification(data.username, {body: data.msg.replace(/<[^>]+>/g, ''), icon: null});
```

Comment:
Since this is inside the chat message handler, you already have access to the div created for the message, so you can use `.text()`

Reply:
Good point.